cobbler/remote.py

```diff
@@ -210,6 +210,19 @@ def runner(self):
             self.remote.api.sync(self.options.get("verbose", False), logger=self.logger)
         return self.__start_task(runner, token, "sync", "Sync", options)
 
+    def background_syncsystems(self, options, token) -> str:
+        """
+        Run a lite Cobbler sync in the background with only systems specified.
+
+        :param options: Not known what this parameter does.
+        :param token: The API-token obtained via the login() method. The API-token obtained via the login() method.
```

Comment:
Duplicate sentence.
Remove the second "The API-token obtained via the login() method."